default.config.js

```diff
@@ -302,6 +302,8 @@ module.exports = {
           }
         }
       },
+      maxScroll: 300,
+      maxScrollDuration: '15m',
```

Comment:
15m is a pretty long time for a default value, 1m should be largely sufficient for most use cases. What do you think?

Reply:
I'm thinking about the `kourou index:dump` action which could take more than 1m for large collections.  
Maybe the admin shouldn't be restrained by theses limitation?

Reply:
Or maybe should make that limit configurable per profile? :thinking:

Reply:
Also, scroll durations can be refreshed each time a new page is requested (see the scrollTTL option), so 1m should really be enough.